draft-birkholz-scitt-architecture.md

```diff
@@ -483,10 +483,10 @@ TS implementations MUST indicate their support for registration policies and MUS
 
 Policy Name | Required `reg_info` attributes | Implementation
 ---|---|---
-TimeLimited | `register_by: uint` | Returns true if now () < register_by at registration time. The ledger MUST store the ledger time at registration along with the claim, and SHOULD indicate it in receipts
+TimeLimited | `register_by: uint` | Returns true if now () < `register_by` at registration time. The ledger MUST store the ledger time at registration along with the claim, and SHOULD indicate it in receipts. The value provided for `register_by` MUST be a Unix timestamp, i.e., a `uint` representing the number of seconds since 00:00:00 UTC, 1 January 1970, without leap seconds.
```

Comment:
there is a cddl type for that (`time`) that maps to CBOR type  `#6.1`, see:
* https://www.rfc-editor.org/rfc/rfc8610.html#appendix-D
*  https://www.rfc-editor.org/rfc/rfc8949.html#name-epoch-based-date-time